### PR TITLE
Add state endpoint

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description:
- added `/state` endpoint
- example response:
<pre><font color="#5FD700">❯</font> curl http://localhost:3000/state -v | jq  <font color="#5F8787">  19:58:37</font>
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 127.0.0.1:3000...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 3000 (#0)
&gt; GET /state HTTP/1.1
&gt; Host: localhost:3000
&gt; User-Agent: curl/7.68.0
&gt; Accept: */*
&gt; 
* Mark bundle as not supporting multiuse
&lt; HTTP/1.1 200 OK
&lt; X-Powered-By: Express
&lt; Content-Type: application/json; charset=utf-8
&lt; Content-Length: 266
&lt; ETag: W/&quot;10a-ux9MnlzEZC7TP0u8CKeRbcJioL8&quot;
&lt; Date: Tue, 11 May 2021 14:13:44 GMT
&lt; Connection: keep-alive
&lt; 
{ [266 bytes data]
100   266  100   266    0     0   259k      0 --:--:-- --:--:-- --:--:--  259k
* Connection #0 to host localhost left intact
<font color="#4D55F5"><b>{</b></font>
<font color="#4D55F5"><b>  </b></font><font color="#839496"><b>&quot;created_users&quot;</b></font><font color="#4D55F5"><b>: {</b></font>
<font color="#4D55F5"><b>    </b></font><font color="#839496"><b>&quot;Alice&quot;</b></font><font color="#4D55F5"><b>: {</b></font>
<font color="#4D55F5"><b>      </b></font><font color="#839496"><b>&quot;password&quot;</b></font><font color="#4D55F5"><b>: </b></font><font color="#659900">&quot;1234&quot;</font><font color="#4D55F5"><b>,</b></font>
<font color="#4D55F5"><b>      </b></font><font color="#839496"><b>&quot;displayname&quot;</b></font><font color="#4D55F5"><b>: </b></font><font color="#659900">&quot;Alice Hansen&quot;</font><font color="#4D55F5"><b>,</b></font>
<font color="#4D55F5"><b>      </b></font><font color="#839496"><b>&quot;email&quot;</b></font><font color="#4D55F5"><b>: </b></font><font color="#659900">&quot;alice@example.org&quot;</font>
<font color="#4D55F5"><b>    },</b></font>
<font color="#4D55F5"><b>    </b></font><font color="#839496"><b>&quot;Brian&quot;</b></font><font color="#4D55F5"><b>: {</b></font>
<font color="#4D55F5"><b>      </b></font><font color="#839496"><b>&quot;password&quot;</b></font><font color="#4D55F5"><b>: </b></font><font color="#659900">&quot;AaBb2Cc3Dd4&quot;</font><font color="#4D55F5"><b>,</b></font>
<font color="#4D55F5"><b>      </b></font><font color="#839496"><b>&quot;displayname&quot;</b></font><font color="#4D55F5"><b>: </b></font><font color="#659900">&quot;Brian Murphy&quot;</font><font color="#4D55F5"><b>,</b></font>
<font color="#4D55F5"><b>      </b></font><font color="#839496"><b>&quot;email&quot;</b></font><font color="#4D55F5"><b>: </b></font><font color="#659900">&quot;brian@example.org&quot;</font>
<font color="#4D55F5"><b>    }</b></font>
<font color="#4D55F5"><b>  },</b></font>
<font color="#4D55F5"><b>  </b></font><font color="#839496"><b>&quot;created_remote_users&quot;</b></font><font color="#4D55F5"><b>: {},</b></font>
<font color="#4D55F5"><b>  </b></font><font color="#839496"><b>&quot;created_groups&quot;</b></font><font color="#4D55F5"><b>: [</b></font>
<font color="#4D55F5"><b>    </b></font><font color="#659900">&quot;HelloGroup&quot;</font><font color="#4D55F5"><b>,</b></font>
<font color="#4D55F5"><b>    </b></font><font color="#659900">&quot;groupname&quot;</font>
<font color="#4D55F5"><b>  ]</b></font>
<font color="#4D55F5"><b>}</b></font>
</pre>

## related issue
- https://github.com/owncloud/owncloud-test-middleware/issues/16

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -62,6 +71,10 @@ function handler(statement) {
         },
     }).then(res => {
         if (res.ok) {
+            // somewhere in the middle of init and cleanup, if current state is to be fetched
+            return fetch("http://localhost:3000/state").then(res => {
+              const state = res.json()
+            })
             return res.text()
```
Comment (dpakach) [inline readme.md:78]: two returns?

File: src/helpers/userSettings.js [hunk 2]
```diff
@@ -12,7 +12,11 @@ const passwords = {
   alt11: process.env.ALT11_USER_PASSWORD || 'E-leven',
```
Comment (dpakach) [inline src/helpers/userSettings.js:12]: please do not change this file. we want this to be in sync with https://github.com/owncloud/web/blob/master/tests/acceptance/helpers/userSettings.js

**Review by individual-it — APPROVED** [review]

File: src/helpers/userSettings.js (hunk 2 above)
Comment (kiranparajuli589) [reply src/helpers/userSettings.js:12]: the existing structure does not remember the states, so the vars are moved globally that way. what do you suggest? do we use another separate file for this?

Comment (dpakach) [reply src/helpers/userSettings.js:12]: it does remembers created users. thats how it's able to delete them at cleanup

Comment (kiranparajuli589) [reply src/helpers/userSettings.js]: not requiring the whole helper file was the problem. fixed

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Pull request merged**